giru/core/repliers.py

```diff
@@ -222,21 +174,13 @@ def text(self):
         return zalgo().zalgofy(choice(self.text_options))
 
 
-def create_replier(pattern, replier_type_value, config):
-    try:
-        replier_type = ReplierType(replier_type_value)
-    except ValueError:  # invalid replier type
-        return OnMatchPatternPickAndSendCorruptedTextMessageReplier(
-            pattern, BAD_CONFIG_SECRET_MESSAGE
-        )
+def create_replier(pattern: str, type: ReplierType, data: list[str]):
+    if type == ReplierType.random_document_from_list:
```

Comment:
Right now if you want to make a replier that has an picture or a video, you have to do it like this, but it doesnt look too good on telegram. 

We could add separate repliers for each type (photo, audio, video, document, etc)

Reply:
![image](https://user-images.githubusercontent.com/1606872/131258180-be1a2ff3-8fe0-4c8d-8676-f3203d2206d0.png)

as you can see, the `photo` looks better than the `document`. I imagine its the same for audio, etc